Kernel/Graphics/Intel/Definitions.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+namespace Kernel::IntelGraphics {
```

Comment:
Can we squash the commit `Kernel/Graphics: Export Intel definitions to a header file`? 

The definitions in Intel/Definitions.h are moved from NativeDisplayConnector.h in ` Kernel/Graphics: Introduce the IntelDisplayConnectorGroup class` into the DisplayConnectorGroup header, and then moved out into a separate file here in `Kernel/Graphics: Export Intel definitions to a header file `